docs/cli/kops_create.md

```diff
@@ -21,7 +22,7 @@ kops create -f FILENAME
 ### Examples
 
 ```
-  # Create a cluster using a cluser spec file
+  # Create a cluster from the configuration in a YAML file
```

Comment:
I would stick with "...from the specification in...".  Or maybe "configuration specification" again?